blocks/tabs/tabs.css

```diff
@@ -29,6 +56,10 @@
     cursor: initial;
 }
```

Comment:
.tabs.vertical .tabs-panel>div:last-of-type {
  flex-basis: 40%
}

.tabs.vertical .tabs-panel>div:first-of-type {
  flex-basis: 60%
}